dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/JDBCDriverService.java

```diff
@@ -746,15 +750,17 @@ public ClassLoader getClassLoaderForLibraryRef() {
     }
 
     /**
-     * Load java.sql.Driver implementations available to the specific class loader and return
-     * the first that accepts the URL that is specified in the vendor properties.
+     * Load java.sql.Driver implementations available to the specific class loader.
+     * If className is specified, return an instance of that class.
+     * Otherwise return the first that accepts the URL.
      *
+     * @param className pre-computed Driver implementation class name to load. This will only ever be available on the DataSourceDefinition path.
```

Comment:
Adding it to DataSourceDefinition first under one pull doesn't mean we aren't going to add it to server config.  The way that DataSourceDefinition className works is that you are expected to supply a vendor implementation name, so this is the most natural approach for supplying a Driver impl to DataSourceDefinition.

Reply:
Yes, wasn't necessarily indicating it needed to be done under this PR, just asking the question generally.  We do need to figure that out soon thought as I believe it will require a message update.